samza-kafka/src/test/scala/org/apache/samza/checkpoint/kafka/TestKafkaCheckpointManager.scala

```diff
@@ -19,22 +19,22 @@
 
 package org.apache.samza.checkpoint.kafka
 
-import kafka.admin.AdminUtils
-import kafka.common.{InvalidMessageSizeException, UnknownTopicOrPartitionException}
-import kafka.message.InvalidMessageException
-import kafka.server.{KafkaConfig, KafkaServer, ConfigType}
-import kafka.utils.{CoreUtils, TestUtils, ZkUtils}
-import kafka.integration.KafkaServerTestHarness
+import _root_.kafka.admin.AdminUtils
+import _root_.kafka.common.{InvalidMessageSizeException, UnknownTopicOrPartitionException}
```

Comment:
What issue are you seeing without _root_?

Reply:
I don't see any. This is done by "optimize import" in Intelij.